tests/scripts/arp_responder.py

```diff
@@ -7,7 +7,12 @@
 import os.path
 from fcntl import ioctl
 from pprint import pprint
-
+import logging
+logging.getLogger("scapy.runtime").setLevel(logging.ERROR)
+import ptf.packet as scapy
+import scapy.all as scapy2
+scapy2.conf.use_pcap=True
+import scapy.arch.pcapdnet
```

Comment:
Can you separate the `logging` and `scapy2` setup steps from the import statements?

Reply:
backlog